Phonebook.Frontend/src/app/pages/page-information/page-information.component.ts

```diff
@@ -10,9 +11,27 @@ import { runtimeEnvironment } from 'src/environments/runtime-environment';
 export class PageInformationComponent implements OnInit {
   public isPreview: boolean = true;
   public runtimeEnvironment: RuntimeEnvironmentInterface = runtimeEnvironment;
-  constructor() {}
+  public contributorsHTML: string = '';
+  constructor(private httpClient: HttpClient) {}
 
   public ngOnInit() {
     this.isPreview = runtimeEnvironment.environment === Environment.production ? false : true;
+    this.loadContributors();
+  }
+
+  public loadContributors(): void {
+    const text = 'Contributors could not be loaded.';
+    this.httpClient
+      .get('https://raw.githubusercontent.com/T-Systems-MMS/phonebook/master/docs/CONTRIBUTORS.md', {
```

Comment:
bitte auf mitgelfieferte assets setzen.

Reply:
Was meinst du damit?^^

Reply:
Das hier keine Abhängigkeit auf externe Systeme sein sollte. Kopier dir Datei in den assets Ordner vom Phonebook und verlinke darauf.

Reply:
Ich habe jetzt mehrere Varianten probiert.. unter anderem ``
`.get('./Frontend.Assets/assets/CONTRIBUTORS.md', { ` 

oder 

`.get('Frontend.Assets/assets/CONTRIBUTORS.md', {
        responseType: 'text'
      })
      .subscribe(success => {
        import('marked').then(marked => {
          this.contributorsHTML = marked.parse(success);
        }),`

Habe auch schon verschiedene Möglichkeiten für die Links probiert, aber nichts hat hingehauen. An was liegt das?

Reply:
..oder meinst du so wie es auch mit der changelog.md gemacht wird? Verstehe auch nicht genau warum wir httpclient nehmen, wenn die Datei ja irgendwo im Repo liegt. Eigentlich müsste man nur den Inhalt der Datei auslesen oder?

Reply:
Müsste so funktionieren. Ist der Ordner richtig?